### ISSUE TITLE
some question about extension data experiment

### ISSUE THREAD

**Issue by Michelleable** [Joshua-Ren/Learning_dynamics_LLM#1]
Thank you for your work and conclusion, I learned a lot of details.

But I sitll have a question about an experiment. At the end of the article, in the comparison between the training of extension data pipeline and the baseline, is there a comparison between 4-epoch data extention and 2-epoch baseline? I am curious if  it is because the squeezing effect can reduce overfitting after it is alleviated, resulting in data extention only having less overfitting than the baseline at the same number of epochs? But in fact, the effect is not as good as the baseline with fewer epochs?

Looking forward to your reply!

**Comment by Joshua-Ren** [comment]
Hi, thanks for this great question. Actually, I have an experiment on 4-epoch baseline vs. 2-epoch data extension (then the total amount of updates during SFT is controlled to be the same). The trend is kind of similar to the current one. But I haven't tried the setting you mentioned (in which the extension case will have x4 more updates than the baseline). But I guess the learning curve of this setting might still be similar to the current version, i.e., argmax slightly drops at the beginning of DPO, which means the squeezing effect is mitigated. Regarding the win rate, I am not so confident which one might perform better, since I think this metric is quite fragile to the selection of the evaluator or the prompting message. That is why I am a bit reluctant to highlight the proposed pipeline in this paper :-) Hope this answers your question to some extent.
